tests/callbacks/test_progress_bar.py

```diff
@@ -558,3 +558,29 @@ def _test_progress_bar_max_val_check_interval(
     total_val_batches = total_val_batches * val_checks_per_epoch
     if trainer.is_global_zero:
         assert trainer.progress_bar_callback.main_progress_bar.total == total_train_batches + total_val_batches
+
+
+def test_progress_bar_redirect(tmpdir):
+
+    """
+    This test ensures the log saved to a file are being trimmed.
+    """
```

Comment:
can we agree on docstring formatting 😅 ? I'm seeing this new pattern emerge recently but why? I thought we had blacken-docs and all that fancy formatting.

Reply:
AFAIK `blacken-docs` is just for Sphinx docs.

@Borda opened https://github.com/PyTorchLightning/pytorch-lightning/pull/8584 for docstring auto-formatting. Not sure if it would fix the extra space before

Reply:
Yes, cleaned out !